### PR TITLE
Sky palace

### PR DESCRIPTION
**Pull request by AardWolf**
Updated for new cheese and Richard interaction, Sky Palace stages and tiles.

### CODE REVIEW

**Comment by AardWolf** [comment]
Please wait for some testing and review before merging!

**Review by DevJackSmith — APPROVED** [review]

**Comment by AardWolf** [comment]
I have done two test submissions, both successful BUT I am not in a palace vault. 

Test with normal situation on lai, as expected
Test with sky pirate swiss on lai, as expected (some pirates for me).

**Review by DevJackSmith — APPROVED** [review]

**Pull request merged**
